src/pages/docker/test/application-testing.md

```diff
@@ -60,10 +60,19 @@ Support for MFTF requires `magento/magento-cloud-docker` version 1.0 or later.
    SELENIUM_HOST=selenium"
    ```
 
+   ```bash
+   CREDENTIALS="magento/MAGENTO_ADMIN_PASSWORD=123123q"
```

Comment:
The Credentials line should probably be in the list above.
The docker command looks duplicated below. Is this intentional?